Pokemon-Showdown/sim/examples/test_random_player.ts

```diff
@@ -15,13 +15,15 @@ const spec = {
 
 const p1spec = {
 	name: "Alice",
-	team: 'Ledian||leftovers|swarm|roost,acrobatics,uturn,toxic|asdfghj|85,85,85,85,85,85||||100|]Swellow||flameorb|guts|protect,bravebird,quickattack,facade||85,85,85,85,85,85||||100|]Malamar||leftovers|contrary|sleeptalk,superpower,rest,knockoff||85,85,85,85,85,85||||100|]Houndoom||houndoominite|flashfire|darkpulse,nastyplot,hiddenpowergrass,fireblast||81,,85,85,85,85||,0,,,,||100|]Victreebel||grassiumz|chlorophyll|powerwhip,knockoff,swordsdance,sleeppowder||85,85,85,85,85,85||||100|]Lugia||leftovers|multiscale|psychic,substitute,toxic,whirlwind||85,,85,85,85,85|N|,0,,,,||100|',
+	team: 'Ledian||leftovers|swarm|roost,acrobatics,uturn,toxic|asdfghj|85,85,85,85,85,85||||100|]Swellow||flameorb|guts|protect,bravebird,quickattack,facade||85,85,85,85,85,85||||10|]Malamar||leftovers|contrary|sleeptalk,superpower,rest,knockoff||85,85,85,85,85,85||||10|]Houndoom||houndoominite|flashfire|darkpulse,nastyplot,hiddenpowergrass,fireblast||81,,85,85,85,85||,0,,,,||10|]Victreebel||grassiumz|chlorophyll|powerwhip,knockoff,swordsdance,sleeppowder||85,85,85,85,85,85||||10|]Lugia||leftovers|multiscale|psychic,substitute,toxic,whirlwind||85,,85,85,85,85|N|,0,,,,||10|',
```

Comment:
What is this change doing?

Reply:
It is changing the levels of every Pokemon except the first ones to 10, which is easier than removing them